tests/conftest.py

```diff
@@ -39,7 +39,8 @@
     uvloop = None  # type: ignore[assignment]
 
 
-pytest_plugins = ("aiohttp.pytest_plugin", "pytester")
+pytest_plugins = ("pytest_asyncio", "pytest_aiohttp", "pytester")
```

Comment:
```suggestion
pytest_plugins = ("pytester",)
```

Reply:
I originally intended to add pytest_aiohttp to enforce it as a required plugin. Might help downstream packagers avoid getting confused by the change.

Reply:
Or we could just add a changelog fragment..

Reply:
Or we could do both...

Reply:
Just add a code comment explaining the motivation.